config/opal_config_asm.m4

```diff
@@ -678,7 +678,7 @@ AC_DEFUN([OPAL_CONFIG_ASM],[
 
             powerpc-*|powerpc64-*|powerpcle-*|powerpc64le-*|rs6000-*|ppc-*)
                 AS_IF([test "$ac_cv_sizeof_long" = "8"],
-                      [atomics_found="PowerPC asssembly"])
+                      [atomics_found="PowerPC assembly"])
```

Comment:
I wonder if this innocent change is causing the build to failure....

I'll *assume* the first failure was due to our nodes being down..but if it happens again, will investigate.

Reply:
It passed main ci...so it should be the former.